src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing` property should be applied to all elements using the universal selector `*`, and it should be included before any other styles are declared.